src/Factory/HalConfigFactory.php

```diff
@@ -6,20 +6,28 @@
 
 namespace ZF\Hal\Factory;
 
-use Zend\ServiceManager\FactoryInterface;
-use Zend\ServiceManager\ServiceLocatorInterface;
+use Interop\Container\ContainerInterface;
+use Zend\ServiceManager\Factory\FactoryInterface;
 
+/**
+ * Class HalConfigFactory
+ *
+ * @package ZF\Hal\Factory
+ */
```

Comment:
Please remove; the information is redundant, and can be gleaned from code reflection already by phpdoc.